src/client/results/mostfrequent/mostFrequentActions.js

```diff
@@ -0,0 +1,23 @@
+export const UPDATE_MOST_USED_WORDS_SEARCH_FILTER = 'UPDATE_MOST_USED_WORDS_SEARCH_FILTER';
+export const updateMostUsedwordsSearch = (filterTerm) => ({
+  type: UPDATE_MOST_USED_WORDS_SEARCH_FILTER,
+  filterTerm,
+});
+
+export const UPDATE_MOST_ACTIVE_USERS_SEARCH_FILTER = 'UPDATE_MOST_ACTIVE_USERS_SEARCH_FILTER';
```

Comment:
:+1: glad to see this as a const now, rather than how I'd made it get dynamically built before